Portfolio/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Develop</h1>
+        <img  height="20px" width="20px" src="../phone.icon.png">

```

Comment:
Consider using a more descriptive alt attribute for image tags to improve accessibility. For example, 'Slack icon'